src/main.js

```diff
@@ -46,7 +46,7 @@ const assetsDir = path.resolve(app.getAppPath(), 'assets');
 // be closed automatically when the JavaScript object is garbage collected.
 var mainWindow = null;
 let spellChecker = null;
-let deeplinkingUrl = null;
```

Comment:
@yuya-oc @jasonblais removing the global variable `deeplinkingUrl` breaks the logic to be able to deep link when the application is closed. Test case would be to have multiple tabs, close the app completely and deep link to a channel in the second tab. It stays on the first tab.

Reply:
@dmeza I believed there are no cases which must refer the global variable though, I’ll check again. Thanks.